kalite/settings/base.py

```diff
@@ -38,20 +38,76 @@
 # Basic setup of logging
 ##############################
 
-# TODO: Use Django's settings.LOGGERS
-
 # Set logging level based on the value of DEBUG (evaluates to 0 if False,
 # 1 if True)
 LOGGING_LEVEL = getattr(local_settings, "LOGGING_LEVEL", logging.INFO)
-LOG = getattr(local_settings, "LOG", logging.getLogger("kalite"))
 
-LOG.setLevel(LOGGING_LEVEL)
-ch = logging.StreamHandler()
-formatter = logging.Formatter('%(asctime)s - %(levelname)s:%(name)s: %(message)s')
-ch.setFormatter(formatter)
-LOG.addHandler(ch)
+# We should use local module level logging.getLogger
+LOG = getattr(local_settings, "LOG", logging.getLogger("kalite"))
 
-logging.getLogger("requests").setLevel(logging.WARNING)  # shut up requests!
+LOGGING = {
```

Comment:
I doubt it. In 0.15, we were logging all static file access.. messages were produced by cherrypy. The messages produced by the additional logging from django.request will be less.

I've opened #4890